lessons/networking/networking-networking-hardware.md

```diff
@@ -1 +1,89 @@
+- Learn what components could be useful to solve connection requirements
+
+# Introduction
+A total of 5 billion people around the world use the internet today based on report from April 2022. Many people are familar with a device called a router that allows them to connect to the internet. Home routers exist in various different forms and in this lesson you will learn what they represent. 
+
+1. [Home Router Components](#home-router-components)
+2. [Switch](#switch)
+3. [Router](#router)
+4. [Broadband Connection](#broadband-connection)
+5. [Wireless Access Point](#wireless-access-point)
+6. [Firewall](#firewall)
+
+# Lesson
+## Home Router Components
+Home routers typically contain following components:
+* Switch (Ethernet LAN)
+* Router (Ethernet WAN)
+* Broadband Connection
+* Wireless Access Point (Wi-Fi)
+* Firewall 
+
+It is important to note that these are typical components, but a router can have more or less depending on the specific application of the device. 
+
+![image](https://user-images.githubusercontent.com/18094862/179405675-5a1c6c1b-4f70-4b8e-89a4-982c522729c4.png)
+
+## Switch
+Switches facilitate sharing of resources on a Local Area Network (LAN) including all of the computers, printers, and servers. All of the connected devices can share information and usually these are all of the devices you own and trust. Ethernet cables are used to connect devices on the network. 
+
+![image](https://user-images.githubusercontent.com/18094862/179407510-eecacf84-1513-4565-a9e4-55acc99d9fc0.png)
+
+## Router
+Routers connect to an Internet Service Provider (ISP) which is part of the Wide Area Network (WAN). ISP connects various routers together so that they can all communicate. 
+
+Depending on the ISP, various different forms broadband connections are used which is described in next section.
+
+ISP networks contain routers just like home routers. The goal of the router is to direct traffic and choose the most efficient route to travel across the network including the internet. 
+
+![image](https://user-images.githubusercontent.com/18094862/179405602-b95cc6ef-db56-4b31-93b6-8354b759b69c.png)
+
+## Broadband Connection
+Broadband connection refers to the high-speed internet access technology which the ISP provides. 
+
+* Cable Modem
+  * Cable modem service uses coaxial cables that deliver picture sand sound to TV.
+* Digital Subscriber Line (DSL)
+  * DSL service uses traditional copper telephone lines.
+* Fiber
+  * Fiber technology uses light to carry data through transparent glass fibers
+* Wireless
+  * Wireless service uses radio link between home location and nearest service provider facility
+* Satellite
+  * Satellite service is a type of wireless broadband, but the communication is with a satellite rather than a nearby facility
+
+## Wireless Access Point
+Wireless Access Point component of the router provides Wi-Fi access to all of the devies on your local network without the need for Ethernet cables. 
```

Comment:
check spelling/grammar.

Introduce some of the physics behind wireless communications. We don't need to go into the math and the theory, but we could introduce concepts such as Frequency and Power. 
I'd like a student to understand the answers to the question
What makes signals stronger and weaker? What can cause interference into a signal?

Reply:
Added this in the update. However, this may be too much for the reader, but I did cover all of the parts you requested.